web/src/app/github/github-organisation-page/github-organisation-page.component.ts

```diff
@@ -10,6 +10,7 @@ import {ActivatedRoute, Router} from '@angular/router';
 })
 export class GithubOrganisationPageComponent implements OnInit {
   orgStats: GithubOrganisationStats[];
+  loading = true;
```

Comment:
it's better to set `true` right before the start fetching

Reply:
make a sense :)